### PR TITLE
Fix test for `dch` capability, add more test cases

### PR DESCRIPTION
**Pull request by dshil**

### CODE REVIEW

File: gits/test/helper.py [hunk 1]
```diff
@@ -378,6 +378,7 @@ def _check_cap_dl(self, n, lines):
                 s = line[0]
                 self._check_string(s, (x, i), (len(s), i))
 
+    @reset_after_executing
```
Comment (eugulixes) [inline gits/test/helper.py:381]: Let's move it to a separate commit which fixes #55.

Comment (dshil) [reply gits/test/helper.py:381]: @eugulixes, fxd

**Review by eugulixes — APPROVED** [review]
